src/main/java/org/generationcp/middleware/util/Util.java

```diff
@@ -451,4 +451,11 @@ public static boolean isLeapYear(final int year) {
 		return Optional.absent();
 	}
 
+	public static int getIntValue(final Integer value) {
```

Comment:
No need for this Util method

Reply:
Util.getIntValue currently used in StudyDataManagerImpl.updateFieldMapTrialInstanceInfo streams 

```
final List<FieldMapLabel> rows = trial.getFieldMapLabels().stream().filter(fieldMapLabel -> Util.getIntValue(fieldMapLabel.getColumn()) > 0).collect(
						Collectors.toList());
					final List<FieldMapLabel> ranges = trial.getFieldMapLabels().stream().filter(fieldMapLabel -> Util.getIntValue(fieldMapLabel.getRange()) > 0).collect(
						Collectors.toList());
```